### ISSUE TITLE
CheckBox rendering incorrect for value=False

### ISSUE THREAD

**Issue by mwx23** [lona-web-org/lona#111]
I think I've found a bug with how the `CheckBox` renders to html. 

It always renders the `checked` attribute.

Expected behaviour would be if `CheckBox()`  or `CheckBox(value=False)` then the `checked` attribute would not be rendered.

```python
from lona.html import CheckBox
c1 = CheckBox()
c2 = CheckBox(value=False)
c3 = CheckBox(value=True)
assert c1.value == False
assert c2.value == False
assert c3.value == True
assert HTML(str(c1))[0].value == False
Traceback (most recent call last):
  File "<input>", line 1, in <module>
AssertionError
assert HTML(str(c2))[0].value == False
Traceback (most recent call last):
  File "<input>", line 1, in <module>
AssertionError
assert HTML(str(c3))[0].value == True
str(c1)
'<input data-lona-node-id="1" type="checkbox" checked="False" data-lona-events="302" />'
str(c2)
'<input data-lona-node-id="2" type="checkbox" checked="False" data-lona-events="302" />'
str(c3)
'<input data-lona-node-id="3" type="checkbox" checked="True" data-lona-events="302" />'
```

**Comment by maratori** [comment]
@mwx23, thanks for reporting.

@fscherf, IMO `TextInput` should not be a base class for `CheckBox`. And it's better to get rid of `TextInput.INPUT_ATTRIBUTE_NAME`. What do you think?

**Comment by fscherf** [comment]
@mwx23, @maratori: the python string representation is only for debug purposes. The real rendering happens in the Javascript client which handles ``checked`` correctly (thats my impression at least).
But we should fix the string representation anyway becase its obviously confusing.

@maratori why do you want to get rid of it?

**Comment by maratori** [comment]
> why do you want to get rid of it?

It is only necessary to be able make `CheckBox` a child of `TextInput`. This inheritance introduces more complexity than solves any problem. If i understand correctly, the bug exists only because of this inheritance.

**Comment by fscherf** [comment]
@maratori I don't think that's the problem. ``checked`` is a boolean property and ``Node.__str__`` just doesn't renders it correctly

**Comment by mwx23** [comment]
Just to be clear this happens when rendered in the browser. All are checked.

```python
from lona import LonaApp, LonaView
from lona.html import HTML, CheckBox

app = LonaApp(__file__)


@app.route('/')
class MyView(LonaView):

    def handle_request(self, request):
        html = HTML(
            CheckBox(),
            CheckBox(value=False),
            CheckBox(value=True),
        )

        return html


app.run(port=8080)

**Comment by fscherf** [comment]
@mwx23 Oh! You are right. It's actually broken on both sides -.- I created a branch and started working on this

**Comment by fscherf** [comment]
@mwx23 Both sides are fixed now. The changes will be available in 1.7.1
